django/db/models/fields/related.py

```diff
@@ -13,6 +13,7 @@
 from django.db.models.deletion import CASCADE, SET_DEFAULT, SET_NULL
 from django.db.models.query_utils import PathInfo
 from django.db.models.utils import make_model_tuple
+from django.db.models.constraints import UniqueConstraint
```

Comment:
This is unused. Please remove it.

Reply:
Oh good catch. I forgot to remove the imports I made in my past tests. Thank you for the review. Fixing it now.